src/main/java/com/rybka/service/exchange/BankAggregatorService.java

```diff
@@ -0,0 +1,46 @@
+package com.rybka.service.exchange;
+
+import com.rybka.constant.BankAggregatorConstants;
+import com.rybka.model.BankData;
+import lombok.Data;
+import lombok.SneakyThrows;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.springframework.stereotype.Service;
+
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@Service
+@Data
+public class BankAggregatorService {
+
+    @SneakyThrows
```

Comment:
set try block and re-throw a custom exception.

Reply:
fixed